packages/libs/eda/src/lib/map/analysis/mapTypes/plugins/BubbleMarkerMapType.tsx

```diff
@@ -100,27 +100,28 @@ function BubbleMapConfigurationPanel(props: MapTypeConfigPanelProps) {
 
   // If the variable or filters have changed on the active marker config
   // get the default overlay config.
-  const activeOverlayConfig = useOverlayConfig({
+  const { overlayConfig, isValidProportion } = useOverlayConfig({
```

Comment:
**active**OverlayConfig didn't seem absolutely necessary so I didn't bother renaming the new object return value

Reply:
That's a holdover from an earlier version of this code base